api/caseWorkerDetailsRouter/mock-error-scenarios.ts

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
Seeing as this is a mock, it should be called `mock-error-scenarios.spec.ts` or else it'll be included in code coverage. Or is this a temporary thing?

Reply:
As discussed, keeping as is due to usage within code and temporary lifespan